.gitmodules

```diff
@@ -1,3 +1,4 @@
-[submodule "Elements"]
-	path = Elements
-	url = https://github.com/lanl/ELEMENTS.git
+[submodule "lib/Elements"]
+	path = lib/Elements
+	url = git@github.com:lanl/ELEMENTS.git
```

Comment:
This could be the problem why I cant recursively clone Elements. Because it used the SSH URL. We should change it to use the HTTPS URL as so
`url = https://github.com/lanl/ELEMENTS.git`